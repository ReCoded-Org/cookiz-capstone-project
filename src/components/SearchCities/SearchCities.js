import { useState } from "react";

const SearchCities = () => {
    const [city, setCity] = useState("");
    return (
        <div className='flex flex-row justify-center'>
            <div className='flex flex-col items-center gap-6 px-8 py-8 '>
                <p className='text-center  text-4xl font-bold'>
                    Places where Cookiez is available
                </p>
                <p className='text-center text-2xl font-normal'>
                    We are currently providing services for limited cities. We
                    plan to expand services across whole country very soon.
                </p>

                <div className='w-2/3 '>
                    <div className='flex flex-row justify-between space-x-3'>
                        <div className='form-group w-full'>
                            <input
                                type='text'
                                className=' w-full p-3 text-2xl font-normal focus-within:border-[#004945] focus:ring-[#004945]'
                                placeholder='Enter a city name'
                                value={city}
                                onChange={(e) => setCity(e.target.value)}
                                required
                            />
                        </div>
                    </div>
                </div>
            </div>
        </div>
    );
};
export default SearchCities;
