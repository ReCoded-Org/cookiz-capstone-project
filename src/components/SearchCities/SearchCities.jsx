const SearchCities = () => {
    return (
        <div className='flex flex-row justify-center'>
            <div className='flex flex-col items-center gap-6 px-8 py-6 '>
                <p className='text-center  text-xl font-bold md:text-3xl'>
                    Places where Cookiez is available
                </p>
                <p className='text-center text-sm font-normal md:text-base'>
                    We are currently providing services for limited cities. We
                    plan to expand services across whole country very soon.
                </p>

                <div className='w-2/3 '>
                    <div className='flex flex-row justify-between space-x-3'>
                        <div className='form-group w-full'>
                            <input
                                type='text'
                                className=' w-full p-3 font-normal focus-within:border-[#004945] focus:ring-[#004945]'
                                placeholder='Enter a city name'
                                required
                            />
                        </div>
                    </div>
                </div>
            </div>
        </div>
    );
};
export default SearchCities;
