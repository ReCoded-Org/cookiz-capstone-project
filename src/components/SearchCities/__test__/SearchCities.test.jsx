import renderer from "react-test-renderer";

import SearchCities from "../SearchCities";

// eslint-disable-next-line no-undef
it("renders correctly", () => {
    const tree = renderer.create(<SearchCities />).toJSON();
    // eslint-disable-next-line no-undef
    expect(tree).toMatchSnapshot();
});
