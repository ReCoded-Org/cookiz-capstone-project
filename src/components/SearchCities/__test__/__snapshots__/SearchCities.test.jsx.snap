// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex flex-row justify-center"
>
  <div
    className="flex flex-col items-center gap-6 px-8 py-8 "
  >
    <p
      className="text-center text-4xl font-bold"
    >
      title
    </p>
    <p
      className="text-center text-2xl font-normal"
    >
      text
    </p>
    <div
      className="w-2/3 "
    >
      <div
        className="flex flex-row justify-between space-x-3"
      >
        <div
          className="form-group flex w-full flex-row gap-3"
        >
          <input
            className=" w-full p-3 text-2xl font-normal focus-within:border-[#004945] focus:ring-[#004945]"
            onChange={[Function]}
            placeholder="placeholder"
            required={true}
            type="text"
            value=""
          />
          <button
            className="w-[200px] bg-primary pt-6 pb-6 text-[24px] text-white hover:bg-green-700"
            onClick={[Function]}
          >
            Enter
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
